.travis.yml

```diff
@@ -31,13 +31,54 @@ script:
   - bazel test --test_output=all //core/... //pull/...
   - bazel test --test_output=all //pull/tests/integration:scrape-test
   - bazel run -c opt //core/tests/benchmark
-  - mkdir _build
```

Comment:
should we keep building with both internal and external deps?

Reply:
That makes it pretty convenient for users to embed civetweb for example. I also think it does not clutter the CMakeLists.txt files too much. For now I'd say we should keep them. Maybe flip the default to use external ones.